.github/integration-in-memory-tests.yml

```diff
@@ -21,6 +23,8 @@ runner-test-matrix:
     runs_on: ubuntu-latest
     triggers:
       - PR Integration CCIP Tests
+      - Nightly Integration CCIP Tests
+      - Run CCIP Integration Tests For Merge Queue
     test_cmd: cd integration-tests/smoke/ccip && go test ccip_fees_test.go -timeout 20m -test.parallel=2 -count=1 -json
```

Comment:
You mentioned that `-test.parallel=2` means that it'll select a runner that may already have a test running on it. Is this understanding correct? Or is this saying that for the chosen runner, we can only run 2 parallel tests at a time?

Reply:
> for the chosen runner, we can only run 2 parallel tests at a time?

This is the correct interpretation. This is also a bit of vestige of older setups, it's likely we don't need this restriction anymore, at least not automatically.